home/translatable_strings.py

```diff
@@ -61,6 +59,5 @@
     'Date must be in this (YYYY-MM-DD) format',
     'Datetime must be in this YYYY-MM-DDTHH:SS format',
     'Date must be in this (YYYY-MM-DD) format',
-    'Datetime must be in this YYYY-MM-DDTHH:SS format',
-    'Need internet access to visit external site.'
+    'Datetime must be in this YYYY-MM-DDTHH:SS format'
```

Comment:
I'm surprised the new strings (e.g. "Comment moderation") don't appear here. It's not crucial though.